app/server.go

```diff
@@ -124,15 +155,15 @@ func RunServer(c *Config) error {
 	tasks := []*scheduler.Task{
 		{
 			CronTab: c.Jobs.FetchResourcesInterval,
-			Func:    jobHandler.FetchResources,
+			Func:    func() error { return jobHandler.FetchResources(context.Background()) },
```

Comment:
Should we use context `WithTimeout` or `WithDeadline` to make sure job is not stucked?

Reply:
skipping it for now as I'm not sure what the duration/deadline value should be

Reply:
timeout could also be passed from config (specified from cli), if not specified, we can make it indefinite. but I think this could be done later on.